ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc

```diff
@@ -181,30 +183,18 @@ TFLiteInterpreter::invoke (const GstTensorMemory * input,
   gint64 start_time = g_get_real_time ();
 #endif
 
-  std::vector <int> tensors_idx;
-  int tensor_idx;
-  TfLiteTensor *tensor_ptr;
   TfLiteStatus status;
   bool size_mismatch = FALSE;
 
-  for (unsigned int i = 0; i < outputTensorMeta.num_tensors; ++i) {
-    tensor_idx = interpreter->outputs ()[i];
-    tensor_ptr = interpreter->tensor (tensor_idx);
-
-    if (tensor_ptr->bytes != output[i].size)
+  for (unsigned int i=0; i < input_tensor_ptr.size(); i++) {
```

Comment:
i = 0?

Reply:
Updated indentation for the changes in the file.